components/atom/spinner/src/index.scss

```diff
@@ -0,0 +1,31 @@
+@import '~@schibstedspain/sui-theme/lib/index';
+
+$bgc-atom-spinner: rgba($c-white, .6);
+$z-atom-spinner: 999;
+
+@mixin spinner-layer {
+  background-color: $bgc-atom-spinner;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  top: 0;
+  z-index: $z-atom-spinner;
+}
+
+.sui-atom-spinner {
+  position: relative;
+
+  &-layer {
+    @include spinner-layer;
```

Comment:
For fixed properties placeholders do the job with less bytes, or 

```sass

&-layer {
   &, &-full {
       @include spinner-layer;
    }
  &-full {
      position: fixed;
   }
}

Reply:
It´s true, because there are no arguments